share/shaders/fx_brightness_contrast.frag

```diff
@@ -0,0 +1,27 @@
+// based on https://www.shadertoy.com/view/XsjSzR
+
+#include "flat_pipeline_interface.glsl"
+#include "fx_input.glsl"
+#include "util.glsl"
+#include "constants.glsl"
+
+const float Brightness = BRIGHTNESS/100.0/2.0;
+const float Contrast = CONTRAST/100.0/4.0;
```

Comment:
```suggestion
const float Contrast = -CONTRAST/100.0/4.0;
```
Inverting the sign of `CONTRAST` should make it behave the same as brightness, i.e. higher values make the screen brighter.

Reply:
I agree it's not intuitive, and there should not be such a noticable change in brightness, but inverting the sign would mean that increasing the contrast value in the UI would effectively _decrease_ the contrast. I'll do some more investigation with the formulas.

Reply:
I changed the implementation based on ImageMagick, which now gives better results and also results in a faster GPU calculation. About the brightness change, that's actually valid - if you change the `Range` value in the shader to `1.0` and change the contrast to `-100`, the resulting image changes to full grayscale, so the visual change in brightness is just a result of the loss of color.